### PR TITLE
Make ecl_lite build for ros2 (on windows 10)

### PR DESCRIPTION
**Pull request by bfjelds**
This is a part of an effort to port Kobuki and Turtlebot2 to ROS2 and Windows 10.

Ported ecl_tools, ecl_core, ecl_lite, & ecl_navigation to ament build system.

### CODE REVIEW

File: ecl_config/include/ecl/config/ecl.hpp [hunk 1]
```diff
@@ -0,0 +1,163 @@
+/**
+ * @file /include/ecl/config/ecl.hpp
```
Comment (bfjelds) [inline ecl_config/include/ecl/config/ecl.hpp]: is this a generated file?  seems like it is and shouldn't be included in the PR.

File: ecl_time_lite/include/ecl/time_lite/config.hpp [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/**
```
Comment (bfjelds) [inline ecl_time_lite/include/ecl/time_lite/config.hpp]: another generated file?

File: ecl_config/include/ecl/config/ecl.hpp (hunk 1 above)
Comment (stonier) [reply ecl_config/include/ecl/config/ecl.hpp]: Yes, looks like you've captured what to do correctly with it now there's no more devel folder (to CMAKE_BINARY_DIR).

**Comment by stonier** [comment]
Thanks!
